css-two-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 40px auto;
+}
+
+.row {
+  display: flex;
+  margin: 15px 0;
+}
+
+.column-full {
+  max-width: 70%;
+  margin: auto;
+}
+
+.column-two {
```

Comment:
make sure to explicitly state the width of the column class, this should be 50%. We don't want display flex setting the width